main/ertp/guide/create-fungible-erights.md

```diff
@@ -15,42 +15,51 @@ House ownership rights are not fungible, they are **specific**.
 
 ### Creating and sending the asset
 
-In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+In ERTP, digital assets are created by a [`mint`](./issuer#mint). Having access to the mint
 gives you the power to create more digital assets of the same type at
 will. For instance, let's say we want to create a new community
 currency called 'BaytownBucks'.
 You would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
 (`npm install @agoric/ertp`) and then:
 
 ```js
-import { makeMint } from '@agoric/ertp';
+import produceIssuer from '@agoric/ertp';
+import harden from '@agoric/harden';
 
-const baytownBucksMint = makeMint('BaytownBucks');
+const { mint: baytownBucksMint, issuer } = produceIssuer('BaytownBucks');
 ```
 
 Great! Now let's use our mint to create 1000 new BaytownBucks.
 
 ```js
-const baytownBucksAssay = baytownBucksMint.getAssay();
-const baytownBucks = baytownBucksAssay.makeUnits;
-const purse = baytownBucksMint.mint(baytownBucks(1000), 'community treasury');
+const baytownBucks = issuer.getAmountMath().make;
+const payment = baytownBucksMint.mintPayment(baytownBucks(1000), 'community treasury');
```

Comment:
We should remove 'community treasury' here, since payment memos/names are no longer a thing